content/end.html

```diff
@@ -12,37 +12,37 @@ <h2> Acknowledgements</h2>
 </p>
 
 <div class="vb"></div>
-
 <p class="full">
-<b>Patrick Lam</b>, director of Software Engineering, for confidentially
-managing the data and unbiasedly advising the project from start to finish.
+<b>Andy Zhang</b> for allowing us to fork the 2018 profile site, saving us time
+and allowing us to focus on making this profile more personal than ever before.
```

Comment:
```suggestion
and letting us focus on making this profile more personal than ever before.
```
(we used allowing already in the previous clause :v